core-ui/src/shared/components/MonacoEditorESM/Editor.js

```diff
@@ -50,6 +55,37 @@ export function Editor({
     memoizedOptions.current = options;
   }
 
+  useEffect(() => {
+    if (editorRef.current) {
+      focusRef.current?.dispose();
+      focusRef.current = editorRef.current.onDidBlurEditorText(() => {
+        setHasFocus(false);
+        if (typeof onBlur === 'function') {
+          onBlur();
+        }
+      });
+      blurRef.current?.dispose();
+      blurRef.current = editorRef.current.onDidFocusEditorText(() => {
+        setHasFocus(true);
+        if (typeof onFocus === 'function') {
+          onFocus();
+        }
+      });
+      return () => {
+        focusRef.current.dispose();
+        blurRef.current.dispose();
+      };
+    }
+  }, [onBlur, onFocus, editorRef]);
+
+  if (editorRef.current) {
+    changeRef.current?.dispose();
+    changeRef.current = editorRef.current.onDidChangeModelContent(() => {
```

Comment:
it installs onDidChangeModelContent listener on every content change, while the same listener is installed in line 150. Can we remove this code?

Reply:
Right, I'll remove it and the blur/focus one.

Reply:
Or NO, if I remove the handler in function near `onMount` making changes in YAML tab won't work...